cmd/end2endtest/censusize.go

```diff
@@ -154,13 +155,19 @@ func (t *E2EMaxCensusSizeElection) Run() error {
 		return err
 	}
 
-	// should not count the last vote
-	firstChoice := fmt.Sprintf("%d", (c.nvotes-1)*10)
-	secondChoice := fmt.Sprintf("%d", 0)
-	resultExpected := [][]string{{firstChoice, secondChoice, "0"}}
+	choices := []*types.BigInt{
+		// should not count the last vote
+		new(types.BigInt).SetUint64(uint64(c.nvotes-1) * 10),
+		// should count the firsts overwrite
+		new(types.BigInt).SetUint64(0),
+		new(types.BigInt).SetUint64(0),
+	}
+
+	// according to the first overwrite
+	expectedResults := [][]*types.BigInt{choices}
```

Comment:
you can merge both assignments 
```go
expectedResults := [][]*types.BigInt{
  new(types.BigInt)....
 }
```

since `choices` is not used anywhere else